apps/events/filters/filters.py

```diff
@@ -103,4 +102,5 @@ def filter_bbox(self, queryset, *lat_lng):
         if min_lat and max_lat and min_lng and max_lng:
             bbox = (float(min_lng), float(min_lat), float(max_lng), float(max_lat))
             area = Polygon.from_bbox(bbox)
-            return queryset.filter(location__within=area)
+            city_queryset = City.objects.filter(location__within=area)
+            return queryset.filter(city_location__in=city_queryset)
```

Comment:
Why do we use two queries here instead of return queryset.filter(location__within=area)?